Rover/AP_Arming_Rover.cpp

```diff
@@ -146,6 +154,14 @@ bool AP_Arming_Rover::arm(AP_Arming::Method method, const bool do_arming_checks)
  */
 bool AP_Arming_Rover::disarm(const AP_Arming::Method method, bool do_disarm_checks)
 {
+    if (method == AP_Arming::Method::RUDDER) {
+        if (rover.g2.motors.active()) {
```

Comment:
this seems to be missing the throttle check that was in the rover code that did rudder input for arming

Reply:
motors active != check for neutral throttle